metis-core/metis-core-service/src/main/java/eu/europeana/metis/core/service/DepublishedRecordService.java

```diff
@@ -27,6 +27,7 @@
 @Service
 public class DepublishedRecordService {
 
+  private static final Pattern PATTERN_PER_LINE = Pattern.compile("\\R");
```

Comment:
Maybe give clearer name? LINE_SEPARATION_PATTERN for instance?

Reply:
Sure